<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Custom Trip</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelNumber">
          <property name="text">
           <string>Select the number of colleges you wish to visit:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="labelTotalDistance">
          <property name="text">
           <string>Total Trip Distance: </string>
          </property>
         </widget>
        </item>
        <item row="10" column="3">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Remove From Cart</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboCollege">
          <property name="maxVisibleItems">
           <number>13</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="startTrip">
          <property name="text">
           <string>Start Trip</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="collegeLocation">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>You are here</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="addCollege">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Add College</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2" colspan="2">
         <widget class="QTableView" name="mainTable"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelCollege">
          <property name="text">
           <string>Select the college you wish to start at:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="spinCollege">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>11</number>
          </property>
          <property name="value">
           <number>11</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="customTrip">
          <property name="text">
           <string>Custom Trip</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="nextCollege">
            <property name="text">
             <string>Next College</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="selectTable">
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextPurchase">
            <property name="text">
             <string>Purchase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="purchaseSouvenir">
            <property name="text">
             <string>Purchase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="souvenirQuantity">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>collegeLocation</zorder>
       <zorder>labelTotalDistance</zorder>
       <zorder>mainTable</zorder>
       <zorder>pushButton</zorder>
       <zorder>labelCollege</zorder>
       <zorder>comboCollege</zorder>
       <zorder>addCollege</zorder>
       <zorder>labelNumber</zorder>
       <zorder>spinCollege</zorder>
       <zorder>customTrip</zorder>
       <zorder>startTrip</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>UCI Trip</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Starting at UCI here is the most efficient path to all the colleges!</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QTableView" name="uciTableView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="uciTripButton">
          <property name="text">
           <string>Start Trip</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="uciTotalDistance"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Souvenirs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="souvenirComboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="souvenirFindButton">
          <property name="text">
           <string>Find Souvenirs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="souvenirTableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Michigan Trip</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
